report.Rmd

```diff
@@ -786,15 +785,15 @@ In this section, a smaller dataset is created and it is verified if the dropout
 
 Replicating  the drop_general variable to see how many instances are dropped generally when the indicator == 1; this means the individual has dropped from experiment
 
-> table(data$drop_general)
+> table(df$drop_general)
```

Comment:
Maybe use `r` code chunk so it is not hardoced but relies on loaded data?

Reply:
Ok, but i have to put the results in as well.. and its a 1 liner code.

i can put in that